### ISSUE TITLE
Weblate automatically updates similar translations

### ISSUE THREAD

**Issue by efdee** [nijel/weblate#960]
I'm getting complaints from our translators that Weblate automatically modifies their translations in some cases. For example, in our project, we have about 10 different translation items whose source is set to "Open", but in French it sometimes translates to "Ouvrir" (to open) and sometimes it translates to "Ouvert" (opened). Now, whenever a translator changes one of these 10 translations to "Ouvert", all the others are changed to "Ouvert" as well, even those that were already translated as "Ouvrir".

I've looked around but can't seem to find a setting to change this behavior. Can you help me?

**Comment by nijel** [comment]
You might want to disable [Allow translation propagation](https://docs.weblate.org/en/latest/admin/projects.html#component-configuration) for these components...

**Comment by efdee** [comment]
Thanks, I missed that option.

It might be nice to have this setting on a project or even systemwide level. I have to update 340 components separately now.

**Comment by nijel** [comment]
You can do the mass update from the Django shell (./manage.py shell):

```python
from weblate.trans.models import SubProject
SubProject.objects.all().update(allow_translation_propagation=False)
```

**Comment by nijel** [comment]
Configurable default value is implemented in 9b50d9a16422a9d4e3a18947329440d43c054064.

**Comment by efdee** [comment]
That's great news, thanks! I'll push an upgrade this week.
